README.md

```diff
@@ -67,28 +65,103 @@ or
 ```
 
 ## Usage:
-```
+
+```shell
+❯❯❯ miqsel --help
```

Comment:
**Required:**
Try to use `#` or `$` to denote shell commands.

Reply:
@sbulage `$` indicate normal user and `#` indicate root user.... I think for python we wont check shell user.... it is applicable for both... you `>>>` is came from fish but its ok... looks great.